polylogue/importers/chatgpt.py

```
@@ -395,13 +414,143 @@ def _copy_attachment(
+            description = description_provider()
+            if description:
+                footnotes[label] = description
+            labels[key] = label
+        return label
+
+    def replacer(match: re.Match[str]) -> str:
+        nonlocal cite_index
+        if cite_index < total_cites:
+            citation = citations[cite_index]
+            cite_index += 1
+            key = ("citation",) + _citation_key(citation)
```

Ruff (0.15.9)

[warning] Consider `("citation", *_citation_key(citation))` instead of concatenation

Replace with `("citation", *_citation_key(citation))`

(RUF005)